src/main/java/com/vestie/vestie/survey/application/SurveyService.java

```diff
@@ -0,0 +1,28 @@
+package com.vestie.vestie.survey.application;
+
+import com.vestie.vestie.member.domain.Member;
+import com.vestie.vestie.member.domain.MemberRepository;
+import com.vestie.vestie.survey.application.dto.SurveyRegisterCommand;
+import com.vestie.vestie.survey.domain.Survey;
+import com.vestie.vestie.survey.domain.SurveyRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class SurveyService {
+
+    private final MemberRepository memberRepository;
+    private final SurveyRepository surveyRepository;
+
+    public Long register(SurveyRegisterCommand command) {
+        Member member = memberRepository.getById(command.memberId());
+        Survey survey = command.toDomain();
+        survey.setMember(member);
```

Comment:
```suggestion
        Survey survey = command.toDomain(member);
```
위처럼 하는것도 깔끔할 것 같은데 어때?

Reply:
매개변수로 받는 방법이 있었구나! setter도 굳이 필요없고 훨씬 깔끔할거 같앙👍🏻